model/browser/events.yaml

```diff
@@ -0,0 +1,104 @@
+groups:
+  - id: event.browser.resource_timing
+    stability: development
+    type: event
+    name: browser.resource_timing
+    brief: >
+      This event describes the timing metrics as provided by PerformanceResourceTiming Performance API.
+      These metrics are related to fetching a resource, such as
+      XMLHttpRequest, Fetch, sendBeacon APIs, SVG, image or script.
+    note: >
+      This event captures data from the [ResourceTiming](https://www.w3.org/TR/resource-timing/).
+      It is recommended to be captured by using the PerformanceResourceTiming API.
+      If the page starts unloading before the page loads,
+      then the partial data can be captured by calling the [performance.getEntriesByType](https://developer.mozilla.org/en-US/docs/Web/API/Performance/getEntriesByType) method.
+    attributes:
+      - ref: url.full
+        requirement_level: required
+        note: >
+          Generally the url doesn't include any query strings or any leading credentials,
+          and it's formatted as scheme://full domain/path only, for example https://user:pwd@www.example.com/path?a=b&c=d
+          would be sent as https://www.example.com/path by both removing any potential PII and reducing the cardinality of the name.
+    body:
+      id: browser.resource_timing
+      requirement_level: required
+      stability: development
+      type: map
+      fields:
+        - id: fetchStart
+          type: double
+          stability: development
+          brief: >
+            The time immediately before the browser starts to fetch the resource,
+            and it is measured in milliseconds.
+          requirement_level: recommended
+          examples: [9.600000000558794]
+        - id: domainLookupStart
+          type: double
+          stability: development
+          brief: >
+            The time immediately before the browser starts the domain name lookup for the resource,
+            and it is measured in milliseconds.
+          requirement_level: recommended
+          examples: [100.6000000014901]
+        - id: domainLookupEnd
+          type: double
+          stability: development
+          brief: >
+            The time immediately after the browser finishes the domain name lookup for the resource,
+            and it is measured in milliseconds.
+          requirement_level: recommended
+          examples: [100.6000000014906]
+        - id: connectStart
+          type: double
+          stability: development
+          brief: >
+             The time immediately immediately before the user agent starts establishing the connection to the server to retrieve the resource,
+             and it is measured in milliseconds.
```

Comment:
```suggestion
             The time immediately before the user agent starts establishing the connection to the server to retrieve the resource, in milliseconds.
```

Reply:
Updated, changes are
1. modified and shortened the brief
2. added deep links
3. changed measurement units from milliseconds to seconds

Reply:
to be discussed: (mentioned by ladd, see details below):
1. measurement units
2. should we consider other potential metrics?

Reply:
resolving, but we can continue some of these discussions below https://github.com/open-telemetry/semantic-conventions/pull/1943#issuecomment-2762382976